src/components/Icon/Icon.tsx

```diff
@@ -19,8 +20,30 @@ const COLORS_WITH_BACKDROPS = [
 // styleguide to generate the props explorer
 interface Props extends IconProps {}
 
+function parseSource(source: string | Function) {
```

Comment:
Put utility functions underneath the component function -show us the important bit first